src/Helpers/DebugMode.php

```diff
@@ -42,11 +42,16 @@ public static function log( string $message ) {
 			return;
 		}
 
-		$date        = date( 'Y-m-d' ); // phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date
-		$time        = date( 'H-i-s' ); // phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date
-		$content_dir = \WP_CONTENT_DIR;
+		$date    = date( 'Y-m-d' ); // phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date
+		$time    = date( 'H-i-s' ); // phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date
+		$log_dir = defined( 'ANTISPAM_BEE_DEBUG_MODE_LOG_DIR' ) ? \ANTISPAM_BEE_DEBUG_MODE_LOG_DIR : \WP_CONTENT_DIR;
```

Comment:
Pretty long const name, would have preferred something a little shorter, e.g. strip the `_MODE`. But we should probably keep it consistent with `ANTISPAM_BEE_DEBUG_MODE_ENABLED`, so I'm fine with either.

Reply:
True, it’s a little long 😬 Because we are not live yet and only two or three people might be using it, I would say let’s change the `ANTISPAM_BEE_DEBUG_MODE_ENABLED` to `ANTISPAM_BEE_DEBUG` and the log to `ANTISPAM_BEE_DEBUG_DIR`?